tools/docker/hue/Dockerfile

```diff
@@ -85,7 +85,10 @@ RUN ./build/env/bin/pip install --no-cache-dir \
   # install after sqlalchemy-clickhouse and version == 1.0.4 \
   # otherwise Code: 516, Authentication failed will display \
   infi.clickhouse_orm==1.0.4 \
-  mysqlclient
+  mysqlclient \
+  PyAthena==2.25.2
+  # PyAthena == 3.x.x is latest, but not working with current configuration \
```

Comment:
Question: Is this because of sqlalchemy 2.0? Its been a while we had worked on sqlalchemy in Hue.